ui/v2.5/src/components/List/Pagination.tsx

```diff
@@ -16,6 +16,11 @@ export const Pagination: React.FC<IPaginationProps> = ({
 }) => {
   const totalPages = Math.ceil(totalItems / itemsPerPage);
 
+  // Build the n-m/total string
+  const firstItemCount:number = Math.min((currentPage-1)*itemsPerPage+1, totalItems);
+  const lastItemCount:number = Math.min(firstItemCount+(itemsPerPage-1), totalItems);
+  const indexText:string = `${firstItemCount}-${lastItemCount}/${totalItems}`;
```

Comment:
`1-40/360` is a bit hard to parse for me. I would prefer `1-40 of 360`.

Reply:
Good call, actually. That's been implemented in 15befa1.